### ISSUE TITLE
Bug: add `uniqueItems: true` in `new-cap`, `no-shadow`, `no-underscore-dangle`, `no-warning-comments`, and `class-methods-use-this` in v11

### ISSUE THREAD

**Issue by lumirlumir** [eslint/eslint#20333]
### Environment

Node version:
npm version:
Local ESLint version:
Global ESLint version:
Operating System:

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```js
<!-- Paste your configuration here -->
```
</details>

```js
<!-- Paste your code here -->
```

### What did you expect to happen?

Working in progress.

### What actually happened?

Working in progress.

### Link to Minimal Reproducible Example

- https://github.com/eslint/eslint/pull/20309
- https://github.com/eslint/eslint/pull/20308
- https://github.com/eslint/eslint/pull/20307
- https://github.com/eslint/eslint/pull/20306

### Participation

- [x] I am willing to submit a pull request for this issue.

### Additional comments

- https://github.com/eslint/eslint/pull/20309
- https://github.com/eslint/eslint/pull/20308
- https://github.com/eslint/eslint/pull/20307
- https://github.com/eslint/eslint/pull/20306

**Comment by lumirlumir** [comment]
I've chose wrong template. So closing.
